addons/mail/static/src/messaging/entity/composer/composer.js

```diff
@@ -228,20 +319,45 @@ function ComposerFactory({ Entity }) {
         _reset() {
             this.update({
                 attachments: [['unlink-all']],
+                mentionedPartners: [['unlink-all']],
                 textInputContent: '',
                 textInputCursorStart: 0,
                 textInputCursorEnd: 0,
             });
         }
 
+        /**
+         * @override
+         */
+        // _update(data) {
+        //     const {
+        //         activeId = this.activeId || 0,
+        //         textInputContent = this.textInputContent || "",
+        //         textInputCursorStart = this.textInputCursorStart || 0,
+        //         textInputCursorEnd = this.textInputCursorEnd || 0,
+        //     } = data;
+        //
+        //     Object.assign(this, {
+        //         activeId,
+        //         textInputContent,
+        //         textInputCursorStart,
+        //         textInputCursorEnd,
+        //     });
+        // }
+
     }
 
     Composer.entityName = 'Composer';
 
     Composer.fields = {
+        activeId: attr({
+            default: 0,
+        }),
```

Comment:
When using an entity id as `attr`, it is usually a sign that it should be a relation instead.

Here, this could be "activePartner" and be a many2one.